frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1251,9 +1252,24 @@ class Worksheet extends React.Component {
         this.setState({ BulkBundleDialog: deleteWorksheetDialog });
     }
 
+    onShowNewUpload = (e) => {
+        e.stopPropagation();
+        this.setState({ uploadAnchor: e.currentTarget });
+        let form = document.querySelector('Menu[id=upload-menu]');
+        Mousetrap.unbind(['up', 'k']);
+        Mousetrap.unbind(['down', 'j']);
+        Mousetrap.unbind(['enter']);
+        Mousetrap.reset();
+        Mousetrap(form).bind(['enter'], function(e) {
+            e.preventDefault();
+            // console.log(e.target.firstElementChild.htmlFor);
+            document.querySelector('label[for=' + e.target.firstElementChild.htmlFor + ']').click();
+        });
```

Comment:
When the user opens the menu, can they press "esc" to exit out of it?

Reply:
yes